### ISSUE TITLE
[FeatureRequest]

### ISSUE THREAD

**Issue by hirvesh** [tamagui/tamagui#178]
Can we have a “Hidden” component similar to what NativeBase [has](https://docs.nativebase.io/hidden)?

It comes with props “from” and “till” which takes breakpoint keys as params and shows/hides elements. Hiding elements do not cause them to render.

**Comment by EfosaSO** [comment]
There's `VisuallyHidden` you can use:

https://tamagui.dev/docs/components/visually-hidden

**Comment by natew** [comment]
Was just going to say ^, though it doesn't have media query support in the same way, and I need to fix its usage with combined space + media query.

**Comment by hirvesh** [comment]
Thanks @natew @EfosaSO - does VisuallyHidden prevent the component from rendering? For e.g. when using breakpoints, I would want for e.g. for some component to render on xs but not lg screen sizes.

This helps prevent unnecessary component mounts depending on screen size 🤔

**Comment by EfosaSO** [comment]
@hirvesh you could combine it with the `[useMedia](https://tamagui.dev/docs/core/use-media)` hook to make you desired `Hidden` in the mean time?

**Comment by hirvesh** [comment]
Absolutely - I can work around this in the meantime!

Much appreciation to you guys for Tamagui, it's pretty awesome work! 🚀

**Comment by EfosaSO** [comment]
![image](https://user-images.githubusercontent.com/22990506/187415573-ef401a2c-ee46-4a25-b07c-f0cc80849c36.png)

Material UI has a implementation prop which you could implement with this to decide if it's CSS only or javascript unmounting

**Comment by natew** [comment]
@EfosaSO Good point, VisuallyHidden could have some sort of strategy eventually.

**Comment by nandorojo** [comment]
Additionally, it would be nice to pass a hidden prop so that it can conditionally get hidden.
